gulpfile.js/tasks/lint.js

```diff
@@ -0,0 +1,36 @@
+const gulp = require('gulp');
+const gulpUtil = require('gulp-util');
+const minimist = require('minimist');
+const spawn = require('child_process').spawn;
+const stylelint = require('stylelint');
+
+/**
+ * Lints the CSS for errors.
+ */
+function _lintCss() {
+  // Pass all command line flags to EsLint.
+  const args = minimist( process.argv.slice( 2 ) );
+  const willFix = args.fix;
+  let willFixFlag = '';
+
+  if (willFix) {
+    willFixFlag = '--fix';
+  }
+
+  return new Promise( ( resolve, reject ) => {
+    spawn(
+      'node_modules/stylelint/bin/stylelint.js',
+      ['client/**/*.scss', 'wagtail/**/static_src/**/scss/**/*.scss', willFixFlag],
+      { stdio: 'inherit' }
+    ).once( 'close', code => {
+      if ( code ) {
+        gulpUtil.log( 'Stylelint exited with code ' + code );
+        reject();
+      }
+      gulpUtil.log( 'Stylelint done!' );
+      resolve();
```

Comment:
So if the command in package.json was `stylelint '**/*.scss' 'wagtail/**/static_src/**/scss/**/*.scss'`, additional arguments can be passed in via `npm run lint:css --fix`, however, it spits out npm ERR! messages if there are linting errors. The recommended approach to squash those in https://medium.com/@bjankord/how-to-lint-scss-with-stylelint-dc87809a9878 is to add `; exit 0` to the end, which has the unfortunate side-effect that command line argument can no longer be passed through, since they'd be going to `exit` instead of `stylelint`. What do you think?